modules/mobile/docs/schemas/ClaimsAndAppealsOverviewAppealsError.yml

```diff
@@ -0,0 +1,40 @@
+type: object
+required:
+  - service
+  - errorDetails
+properties:
+  service:
+    type: string
+    enum:
+      - appeals
+      - claims
+  errorDetails:
+    type: array
+    items:
+      type: array
+      description: Array of arrays from failing upstream services. Used for debugging only.
```

Comment:
I'm not sure if this is accurate. Looking at an example from the specs, I think errorDetails is an array of objects, not an array of arrays. I think you may have an extra layer of depth in the docs than exists in the response. 
```
"meta"=>
  {"errors"=>
    [{"service"=>"appeals",
      "errorDetails"=>
       [{"title"=>"Bad Gateway",
         "detail"=>"Received a 500 response from the upstream server",
         "code"=>"CASEFLOWSTATUS500",
         "source"=>"No method error",
         "status"=>"502"}]}],
```

Reply:
Oo thanks for catching that. I just edited the original documentation which had the double nested arrays but I should have noticed that difference